iprange.go

```diff
@@ -0,0 +1,45 @@
+package netaddr
+
+import (
+	"fmt"
+	"net"
+)
+
+// IPRange range of ips not necessarily aligned to a power of 2
+type IPRange struct {
+	First, Last net.IP
+}
+
+func (r *IPRange) String() string {
+	return fmt.Sprintf("[%s,%s]", r.First, r.Last)
+}
+
+// IPRangeFromIPNet get an IPRange from an *ip.Net
+func IPRangeFromIPNet(cidr *net.IPNet) *IPRange {
+	return &IPRange{
+		First: cidr.IP,
```

Comment:
I'm thinking that we merge https://github.com/IBM/netaddr/pull/16 and then update this to call `NetworkAddr(cidr)`. What do you think?